src/leo-client-app/src/pages/satellite-logs/index.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
```

Comment:
what does this mean

Reply:
rends client side so you can use hooks iirc